plugins/chunter-resources/src/components/Message.svelte

```diff
@@ -83,6 +82,16 @@
 
   $: isEditing = false
 
+  let extensions: Ref<Mixin<ChunterMessageExtension>>[] = []
```

Comment:
Should ObjectPresenter works fine with Hiearchy.findMixinMixin?

Reply:
I am not sure I understand, but the `ObjectPresenter` in this case showing presenter for main message class and not for the presenter.